sdk/ai/azure-ai-inference/pom.xml

```diff
@@ -85,5 +85,22 @@
       <version>1.0.0</version> <!-- {x-version-update;com.knuddels:jtokkit;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-monitor-opentelemetry-exporter</artifactId>
+      <version>1.0.0-beta.31</version> <!-- {x-version-update;com.azure:azure-core-tracing-opentelemetry;dependency} -->
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-tracing-opentelemetry</artifactId>
+      <version>1.0.0-beta.52</version> <!-- {x-version-update;com.azure:azure-core-tracing-opentelemetry;dependency} -->
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-exporter-otlp</artifactId>
+      <version>1.44.1</version>  <!-- {x-version-update;io.opentelemetry:opentelemetry-exporter-otlp;dependency} -->
+    </dependency>
```

Comment:
discuss with Liudmila - is it okay to have these opentelemetry dependencies for the samples in test scope?